mail_message_icon/wizard/mail_resend_message_views.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+  <data>
+    <record id="mail_resend_message_view_form_inherit" model="ir.ui.view">
```

Comment:
ID does not follow OCA Guidelines (see comment in `views/mail_message_views.xml`)